h2/src/main/org/h2/expression/function/TableFunction.java

```diff
@@ -17,26 +17,25 @@
 import org.h2.table.Column;
 import org.h2.value.Value;
 import org.h2.value.ValueArray;
+import org.h2.value.ValueInt;
 import org.h2.value.ValueNull;
 import org.h2.value.ValueResultSet;
 
 /**
  * Implementation of the functions TABLE(..) and TABLE_DISTINCT(..).
```

Comment:
and UNNEST

Reply:
Yes, I missed that. I'll add it to javadoc later.